zeek/plugin/src/plugin.cc

```diff
@@ -389,21 +391,26 @@ void plugin::Zeek_Spicy::Plugin::Done() {
     hilti::rt::done();
 }
 
+void plugin::Zeek_Spicy::Plugin::loadModule(const hilti::rt::filesystem::path& path) {
+    try {
+        ZEEK_DEBUG(hilti::rt::fmt("Loading %s", path.native()));
+
+        if ( ! _libraries.count(path) ) {
+            _libraries.insert({path, hilti::rt::Library(path)});
+            if ( auto load = _libraries.at(path).open(); ! load )
+                hilti::rt::fatalError(hilti::rt::fmt("could not open library path %s: %s", path, load.error()));
+        }
+    } catch ( const hilti::rt::EnvironmentError& e ) {
```

Comment:
Why are we catching this specific exception here? Could we catch `Exception` instead?

Reply:
Not sure, that's existing code that I just moved. I assumed it was due to the library loading in some way?

Reply:
I checked and it looks like nothing in the loading throws this exception today. This pattern was probably introduced because of this review comment: PRhttps://github.com/zeek/spicy/pull/317#pullrequestreview-414229770-body-html-edit-form (the comment on `library.cc`); previously we caught `std::runtime_error`. 

The same pattern is copied in `hilti/toolchain/src/compiler/driver.cc`.